packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -260,11 +260,13 @@ class DropdownMenu<T> extends StatefulWidget {
 
   /// Determine if the on-screen virtual keyboard shows on a touch event.
   ///
-  /// defaults to true. If false, the text field will not request focus
-  /// when it is tapped and the virtual keyboard will not show up.
-  /// This is useful on mobile platforms when the options are limited
-  /// and searching is unnecessary.
-  final bool showVirtualKeyboard;
+  /// If this is null or false, the text field will not request focus when it is tapped,
+  /// and the virtual keyboard will not show up on mobile platforms. This is useful
+  /// when the options are limited and searching is unnecessary. If this is set to true,
+  /// the text field is focusable and the virtual keyboard will show when the text field
+  /// is tapped. On desktop platforms, this value will be ignored and the text field will
```

Comment:
Instead of `showVirtualKeyboard` maybe this should be `takesFocus`, since one might set it to false on desktop platforms to prevent searching or filtering?

@gspencergoog - do we have a conventional name for a flag that means "request the focus when clicked or tapped"?

Reply:
No, we don't have a set name for that, since text fields are the only thing that do that.

`takesFocus` doesn't include the part about needing user interaction to take focus, it sounds like a synonym for `canRequestFocus` which is what we currently use for the ability to take focus.

Maybe `requestFocusOnTap` instead?

But the docs here say "If set to true ... the text field is focusable", implying that if it is set to false, then the text field is not focusable, which I don't think is the case, correct? It's still focusable, it just doesn't request focus on tap, meaning that you could tab to get to it, and it would focus, you just can't tap on it to focus it, am I right?

Also, it seems odd to have it be ignored on some platforms and not others.  It seems like desktop platforms could just default to false, and mobile default to true.

Reply:
`requestFocusOnTap` sounds better and agreed that the default value would be false for mobile, true otherwise.

Reply:
> No, we don't have a set name for that, since text fields are the only thing that do that.
> 
> `takesFocus` doesn't include the part about needing user interaction to take focus, it sounds like a synonym for `canRequestFocus` which is what we currently use for the ability to take focus.
> 
> Maybe `requestFocusOnTap` instead?
> 
> But the docs here say "If set to true ... the text field is focusable", implying that if it is set to false, then the text field is not focusable, which I don't think is the case, correct? It's still focusable, it just doesn't request focus on tap, meaning that you could tab to get to it, and it would focus, you just can't tap on it to focus it, am I right?
> 
> Also, it seems odd to have it be ignored on some platforms and not others. It seems like desktop platforms could just default to false, and mobile default to true.

`requestFocusOnTap` sounds good to me! Thanks for the suggestion. And yes, I think "focusable" here is not accurate. I need to change the doc. 

And will set the default value to false for mobile platforms and true for desktop platforms. Thanks!